pkg/transformer/openshift/openshift.go

```diff
@@ -338,7 +339,15 @@ func (o *OpenShift) Transform(komposeObject kobject.KomposeObject, opt kobject.C
 	buildRepo := opt.BuildRepo
 	buildBranch := opt.BuildBranch
 
-	for name, service := range komposeObject.ServiceConfigs {
+	// Need to ensure the OpenShift objects are in a consistent order
+	var sortedKeys []string
+	for name := range komposeObject.ServiceConfigs {
+		sortedKeys = append(sortedKeys, name)
+	}
+	sort.Strings(sortedKeys)
```

Comment:
can we create a function out of this so that we can use it in both kubernetes and openshift?